plugins/apache_druid_logs.yaml

```diff
@@ -0,0 +1,296 @@
+version: 0.0.1
+title: Apache Druid
+description: Log parser for Apache Druid
+parameters:
+  - name: enable_broker_logs
+    description: Enable to collect Apache Druid broker logs
+    type: bool
+    default: true
+  - name: broker_log_path
+    description: Absolute filepath containing Apache druid broker logs. Required if enable_broker_logs is true
+    type: "string"
+  - name: enable_coordinator_overlord_logs
+    description: Enable to collect Apache Druid coordinator and overlord logs
+    type: bool
+    default: true
+  - name: coordinator_overlord_log_path
```

Comment:
Should these all be marked as `required: true` and have a `relevant_if` block for only relevant if their `enabled_` is true? Probably a question for BPOP team for what the standards are.

Reply:
Yeah thats how we usually do this sort of thing, you'd want

```yaml
relevantIf:
        - name: enable_coordinator_overlord_logs
          operator: equals
          value: "regex"
required: true
```

Reply:
yes that's usually how we try to do these, is showing only relevant. Starting to get some standards docs up (destinations and sources incoming eventually): https://github.com/observIQ/engineering-standards/blob/main/bindplane-development/processor-standards.md. but not seeing anything related to relevantifs on there yet. I'll bring it up in our code quality meeting.